api/src/pcapi/admin/custom_views/boost_pivot_view.py

```diff
@@ -0,0 +1,133 @@
+import typing
+
+from flask import flash
+from flask_admin.form import SecureForm
+from werkzeug.exceptions import Forbidden
+from wtforms import Form
+from wtforms import IntegerField
+from wtforms import PasswordField
+from wtforms import StringField
+from wtforms.validators import DataRequired
+from wtforms.validators import URL
+
+from pcapi.admin.base_configuration import BaseAdminView
+import pcapi.core.offerers.models as offerers_models
+import pcapi.core.providers.models as providers_models
+from pcapi.core.providers.repository import get_provider_by_local_class
+from pcapi.models import db
+
+
+class BoostPivotForm(SecureForm):
+    venue_id = IntegerField("Identifiant lieu (pass Culture)", [DataRequired()])
+    cinema_id = IntegerField("Identifiant Cinéma (Boost)", [DataRequired()])
+    username = StringField("Nom d'utilisateur (Boost)", [DataRequired()])
+    password = PasswordField("Mot de pass (Boost)", [DataRequired()])
```

Comment:
```suggestion
    password = StringField(("Mot de passe (Boost)", [DataRequired()])
```
Je ne suis pas sûr que l'on y gagne quelque chose à traiter ce champ comme mot de passe, mais par contre le navigateur ouvre une modale d'enregistrement de mot de passe, et le contenu du champ est effacé lors du rechargement.